nodes can only be a member of a single machineconfigpool (#27797)
openshift/openshift-docs: modules/machine-config-overview.adoc

features needed by an application. However, examples in this section focus
on changes to the default pool types.

[IMPORTANT]
====

> A node can have multiple labels applied that indicate it is "master", "worker", etc,

Reviewer: > 🚫  etc. n
Do not use. Use “and so on” when you list a clear sequence of elements, such as “1, 2, 3, and so on” or “Monday, Tuesday, Wednesday, and so on.”

Since this isn't a clear sequence, maybe:
```suggestion
A node can have multiple labels applied that indicate its type, such as `master` or `worker`,
```

> however it can only be a member of a *single* MachineConfigPool.

Reviewer: ```suggestion
however it can be a member of only a *single* machine config pool.
```
We're in the middle of changing the way that we [refer to API objects](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting), so since this isn't a specific reference, it should be split up.
